pkg/controllers/node/controller.go

```diff
@@ -154,9 +129,11 @@ func (c *Controller) Builder(ctx context.Context, m manager.Manager) operatorcon
 			}),
 		).
 		Watches(&source.Channel{Source: ch}, &handler.EnqueueRequestForObject{}).
-		WithOptions(controller.Options{MaxConcurrentReconciles: 10})
-}
-
-func (c *Controller) LivenessProbe(_ *http.Request) error {
-	return nil
+		WithEventFilter(predicate.NewPredicateFuncs(func(obj client.Object) bool {
+			_, ok := obj.GetLabels()[v1alpha5.ProvisionerNameLabelKey]
+			return ok
+		})).
+		WithEventFilter(predicate.NewPredicateFuncs(func(obj client.Object) bool {
+			return obj.GetDeletionTimestamp().IsZero()
+		})))
```

Comment:
We can punt on this change, but this doesn't change the behavior of the controller. This filters the function before doing a `Get` against the cache, which is best-practice to avoid unnecessary reconciles.

Reply:
The checking logic is the same as having it as the first thing in the reconcile loop.